dynamic_rest/filters.py

```diff
@@ -203,10 +203,9 @@ def _extract_filters(self, **kwargs):
 
         """
```

Comment:
@rgangopadhya - Since you wanted to use this as a standalone function, I wonder if we should just move it to a helper function and make this method wrap around the function.

Something like:

```python
def extract_filters(view=None, filters_map=None):
```